@xen-orchestra/backups/formatVmBackups.mjs

```diff
@@ -25,6 +25,9 @@ function formatVmBackup(backup) {
       name_description: backup.vm.name_description,
       name_label: backup.vm.name_label,
     },
+    // any backup with at least one differential disk is a delta
+    differential:
```

Comment:
`hasDifferencingVhds`?

Reply:
IMHO, it's better to have the number of differencing and the number of dynamic VHDs.